src/index.js

```diff
@@ -297,6 +297,7 @@ import {
         const announcement = new Announcement(config, room, election, rescraper);
         announcement.setRescraper(rescraper);
         announcement.initAll();
+        rescraper.setAnnouncement(announcement);
```

Comment:
This is what was my original intention that for some reason didn't make it to the final version of the PR - fixing it here